src/InfoBoxes/Content/Other.cpp

```diff
@@ -188,19 +188,13 @@ UpdateInfoBoxNbrSat(InfoBoxData &data) {
     const NMEAInfo &basic = CommonInterface::Basic();
     const GPSState &gps = basic.gps;
 
-    StaticString<80> Temp;
-    StaticString<20> GpsStatus;
-
-    GpsStatus = gettext(GetGPSStatus(basic));
-    data.SetComment(_(GpsStatus));
+    data.SetComment(_(GetGPSStatus(basic)));
```

Comment:
Using `_()` is wrong here, because this shortcut is only meant for string literals, to be collected into the PO template. `gettext()` is the function for non-literal strings.

Reply:
ok, well understood.